packages/sample-apps/windows/SampleAppCPP/ReactPackageProvider.cpp

```diff
@@ -15,7 +15,7 @@ namespace winrt::SampleAppCpp::implementation {
 //===========================================================================
 
 void ReactPackageProvider::CreatePackage(IReactPackageBuilder const &packageBuilder) noexcept {
-  AddAttributedModules(packageBuilder);
+  AddAttributedModules(packageBuilder, true);
```

Comment:
do we still have samples (or actually more asking about tests in CI) that exercise the nativemodule case?

Reply:
NativeModules have fair amount of UTs here:
https://github.com/microsoft/react-native-windows/blob/main/vnext/Microsoft.ReactNative.Cxx.UnitTests/NativeModuleTest.cpp